src/patternfly/components/DataList/data-list.scss

```diff
@@ -2,15 +2,25 @@ $pf-data-list--breakpoint--mobile: 380px;
 
 .pf-c-data-list {
   --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-data-list--BorderTopColor: var(--pf-global--Color--light-200);
```

Comment:
This is a `color` var. Can you use `--pf-global--BorderColor--light-200` instead?